src/helpers/fix.spec.js

```diff
@@ -29,16 +29,16 @@
 import {expect} from 'chai';
```

Comment:
Whole file should be converted to use fixugen

Reply:
Refactored tests for fix.js to use fixugen

[86b9ae0](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/86b9ae07ece0a3deea7d55cc10f0661b3340141e) and [e23304a](https://github.com/NatLibFi/melinda-rest-api-commons/pull/40/commits/e23304a921e5971b57219bb51e2dce7abb078c42)